Cabal/Distribution/Compat/ResponseFile.hs

```diff
@@ -47,8 +50,18 @@ unescape args = reverse . map reverse $ go args NoneQ False [] []
 #endif
 
 expandResponse :: [String] -> IO [String]
-expandResponse = fmap concat . mapM expand
+expandResponse = go recursionLimit "."
   where
-    expand :: String -> IO [String]
-    expand arg@('@':f) = unescapeArgs <$> readFile f `catchIOError` (const $ return arg)
-    expand x = return [x]
+    recursionLimit = 100

```

Comment:
`libiberty` uses 2000, but I'm okay with using 100.

Reply:
I think they expand iteratively, so the 2000 is the total number of response files encountered in their expansion tree, whereas this 100 is tree depth.